lib/srv/discovery/fetchers/aws-sync/ec2.go

```diff
@@ -61,10 +61,10 @@ func (a *awsFetcher) fetchAWSEC2Instances(ctx context.Context) ([]*accessgraphv1
 		errs    []error
 	)
 	eG, ctx := errgroup.WithContext(ctx)
-	// Set the limit to 10 to avoid too many concurrent requests.
+	// Set the limit to 5 to avoid too many concurrent requests.
 	// This is a temporary solution until we have a better way to limit the
 	// number of concurrent requests.
-	eG.SetLimit(10)
+	eG.SetLimit(5)
```

Comment:
1. Should we make this configurable?
2. Does AWS has different rate limits for different accounts? If yes, this change will help some and make stuff slower for other.

Reply:
1. No
2. Yes and it depends if you requested an API quota increase. Once we move to  SDK V2, I can implement a proper retry and rate limiter but until then we are kind of limited on what we can do.